osf_pigeon/pigeon.py

```diff
@@ -171,6 +184,44 @@ def main(
         )
         modify_metadata_with_retry(ia_item, metadata)
 
+        return ia_item.urls.details
+
+
+def gather_children(
+        guid,
+        datacite_username=settings.DATACITE_USERNAME,
+        datacite_password=settings.DATACITE_PASSWORD,
+        datacite_prefix=settings.DATACITE_PREFIX,
+        ia_access_key=settings.IA_ACCESS_KEY,
+        ia_secret_key=settings.IA_SECRET_KEY):
+
+    response = get_with_retry(
+        f'{settings.OSF_API_URL}v2/registrations/?filter[root]={guid}',
+        retry_on=(429,))
+    children = response.json()['data']
+
+    children_guids = [child['id'] for child in children if child['id'] != guid]
+
+    children_data = []
+
+    for child_guid in children_guids:
+        child_data = {}
+        child_url = main(
+            child_guid,
+            datacite_username=datacite_username,
+            datacite_password=datacite_password,
+            datacite_prefix=datacite_prefix,
+            ia_access_key=ia_access_key,
+            ia_secret_key=ia_secret_key,
+            multi_level=False
```

Comment:
This looks like it's getting only the immediate children, Just want to confirm this is the correct behavior.